ras/ras_ppcutils.py

```diff
@@ -291,10 +291,22 @@ def test_ls_vdev(self):
         """
         self.log.info("===============Executing ls-vdev tool test===="
                       "===========")
+        self.is_fail = 0
         self.run_cmd("ls-vdev")
         self.run_cmd("ls-vdev -h")
         self.run_cmd("ls-vdev -V")
-        self.error_check()
+        dev_name = self.run_cmd_out("ls-vdev").split()[1]
+        flag = True
+        for line in self.run_cmd_out("lsblk").splitlines():
+            if dev_name in line:
+                if dev_name == line.split()[0]:
+                    flag = False
+                    break
+        if flag:
+            self.is_fail += 1
+        if self.is_fail >= 1:
+            self.fail("%s command(s) failed in ls-vdev tool "
+                      "verification" % self.is_fail)
```

Comment:
`self.error_check()` is this not going to help in this scenario to validate?

Reply:
No this is not going to help here..